packages/ruleset/test/rules/api-symmetry.test.js

```diff
@@ -756,6 +756,59 @@ describe(`Spectral rule: ${ruleId}`, () => {
       expect(results).toHaveLength(0);
     });
 
+    it('prototype schema is identical to canonical schema (no omitted properties)', async () => {
+      const testDocument = makeCopy(rootDocument);
+
+      testDocument.components.schemas.DrinkPrototype =
+        testDocument.components.schemas.Drink;
+
+      const results = await testRule(ruleId, rule, testDocument);
+      expect(results).toHaveLength(0);
+    });
+
+    it('prototype schema is identical to canonical - properties nested in oneOf', async () => {
+      const testDocument = makeCopy(rootDocument);
+
+      testDocument.components.schemas.Soda.properties.id = {
+        type: 'string',
+      };
+
+      testDocument.components.schemas.Juice.properties.id = {
+        type: 'string',
+      };
+
+      const results = await testRule(ruleId, rule, testDocument);
+      expect(results).toHaveLength(0);
+    });
+
+    it('prototype schema is identical to canonical but defines writeOnly properties', async () => {
+      const testDocument = makeCopy(rootDocument);
+
+      testDocument.components.schemas.DrinkPrototype = makeCopy(
+        testDocument.components.schemas.Drink
+      );
+
+      testDocument.components.schemas.DrinkPrototype.properties.random = {
+        type: 'string',
+        description: 'the only differentiator',
+        writeOnly: true,
+      };
+
+      const results = await testRule(ruleId, rule, testDocument);
+      expect(results).toHaveLength(0);
+    });
+
+    it('patch schema is identical to canonical schema', async () => {
+      const testDocument = makeCopy(rootDocument);
+
+      testDocument.components.schemas.CarPatch.properties.id = {
+        type: 'string',
+      };
+
+      const results = await testRule(ruleId, rule, testDocument);
+      expect(results).toHaveLength(0);
+    });
```

Comment:
These were all negative tests before - I simply edited them to be positive tests now.